.travis.yml

```diff
@@ -2,14 +2,12 @@ language: java
 
 sudo: false
 
-script:
-  - jdk_switcher use oraclejdk7
-  - mvn clean verify -P jdk7
-  - jdk_switcher use oraclejdk8
-  - mvn clean verify -P jdk8
+jdk:
+  - oraclejdk8
+  - oraclejdk7
 
-after_success:
-  - mvn clean test jacoco:report coveralls:report
+script:
```

Comment:
> I see three different checks on the PR, one for the build of the branch, one for the merge (?) and one for the coverage.

How is this configured currently, am I only allowed to merge when all checks pass? Then it would be nice to directly see the distinction between failing tests and missing coverage.

Regarding the merge build, travis seems to do the same as we do in stash. Not sure how this works with github, there must also be some kind of special remote ref. I only noticed this when looking at the issue with the tracer autoconfiguration.